### PR TITLE
Fix ChunkLoadError

### PR DESCRIPTION
**Pull request by pavelgrinchenko**
- Update Gatsby.js to the latest version for getting [this fix](https://github.com/gatsbyjs/gatsby/pull/18051). It allows Gatsby.js ignore ChunkLoadError and load static pages when it happens.
- Ignore ChunkLoadError in Sentry
- Optimize images loading checker a little bit

### CODE REVIEW

**Comment by shcheklein** [comment]
@pavelgrinchenko thanks!

1. Let's resolve conflicts
2. See some comments I left

File: gatsby-config.js [hunk 1]
```diff
@@ -70,7 +70,13 @@ const plugins = [
     options: {
       dsn: process.env.SENTRY_DSN,
       environment: process.env.NODE_ENV,
-      enabled: process.env.NODE_ENV === 'production'
+      release: process.env.CIRCLE_SHA1,
+      enabled: process.env.NODE_ENV === 'production',
+      ignoreErrors: [
+        // Ignore errors on missing chunks when new version was deployed
```
Comment (shcheklein) [inline gatsby-config.js]: Please, let's put a better comment here - it's not clear why do we have this.

File: gatsby-config.js [hunk 2]
```diff
@@ -70,7 +70,13 @@ const plugins = [
...
+        'ChunkLoadError'
+      ],
+      ignoreUrls: [/extensions\//i, /^chrome:\/\//i]
```
Comment (shcheklein) [inline gatsby-config.js]: why do we need this?

File: gatsby-config.js (hunk 1 above)
Comment (pavelgrinchenko) [reply gatsby-config.js]: Updated comment. Now it should me more clear. Wdyt?

File: gatsby-config.js (hunk 2 above)
Comment (pavelgrinchenko) [reply gatsby-config.js]: Added comment to the code

**Pull request merged**